astroimtools/cutout_tools.py

```diff
@@ -132,7 +132,7 @@ def make_cutouts(catalogname, imagename, image_label, apply_rotation=False,
 
             try:
                 cutout_arr = reproject_interp(
-                    (data, wcs), cutout_hdr, shape_out=(y_pix, x_pix), order=2)
+                    (data, wcs), cutout_hdr, shape_out=(int(np.ceil(y_pix)), int(np.ceil(x_pix))), order=2)
```

Comment:
If you want a `round` function that works the same in Python 2 and 3, you can use:

math.floor(x + math.copysign(0.5, x))